morpheus/_lib/include/morpheus/messages/control.hpp

```diff
@@ -166,6 +166,37 @@ class TensorMemory;
 // System-clock for better compatibility with pybind11/chrono
 using time_point_t = std::chrono::time_point<std::chrono::system_clock>;
 
+class PythonByteContainer : public std::vector<uint8_t>
+{
+  public:
+    PythonByteContainer() = default;
+    PythonByteContainer(pybind11::object py_obj) : m_py_obj(std::move(py_obj)) {}
+
+    pybind11::object get_py_obj() const
+    {
+        return m_py_obj;
+    }
+
+  private:
+    pybind11::object m_py_obj;
```

Comment:
We will want to change `pybind11::object` here to the `PyHolder` class in `pymrc`. `PyHolder` wraps a `pybind11::object` and makes sure that we have the GIL when its destroyed (which would throw an error otherwise`.